development/extensions/index.md

```diff
@@ -284,9 +355,169 @@ Once installed on the [Onboard Computer](@/hardware/required/onboard-computer/in
 
 ## Development Process
 
+[Detailed flowchart](https://github.com/bluerobotics/ardusub-zola/blob/BlueOS-1.1/development/extensions/development-process.mermaid)
+
+### Expected Development Cycle
+
+0. (_If possible_) create and test initial backend functionality within BlueOS
+1. Wrap Extension functionality into a Docker Image
+1. Upload to Docker Hub
+1. Test locally (using a "manual"/development install)
+1. Submit the extension to the Bazaar
+1. Get it accepted
+1. Install it from the store
+1. Test locally
+1. Make improvements
+1. Upload a new version
+1. Repeat from step 6.
+
```

Comment:
Should add https://github.com/bluerobotics/BlueOS/issues/1219 into this ?

Reply:
Documentation should never be ahead of the available features.

Once the feature is available then it definitely should be documented though :-)